services/storage/src/simcore_service_storage/api/v0/openapi.yaml

```diff
@@ -10,84 +10,82 @@ info:
+        - datasets
       summary: Get datasets metadata
       description: returns all the top level datasets a user has access to
       operationId: get_datasets_metadata
       parameters:
-      - required: true
-        schema:
-          type: integer
-          title: Location Id
-        name: location_id
-        in: path
-      - required: true
-        schema:
-          type: integer
-          exclusiveMinimum: true
-          title: User Id
-          minimum: 0
-        name: user_id
-        in: query
+        - required: true
+          schema:
+            type: integer
+            title: Location Id
+          name: location_id
+          in: path
+        - required: true
+          schema:
+            type: integer
+            exclusiveMinimum: true
+            title: User Id
+            minimum: 0
+          name: user_id
+          in: query
       responses:
         '200':
           description: Successful Response
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Envelope_list_simcore_service_storage.models.DatasetMetaData__'
-  /v0/locations/{location_id}/datasets/{dataset_id}/metadata:
+  '/v0/locations/{location_id}/datasets/{dataset_id}/metadata':
     get:
       tags:
-      - datasets
+        - datasets
       summary: Get Files Metadata
       description: returns all the file meta data inside dataset with dataset_id
       operationId: get_files_metadata_dataset
       parameters:
-      - required: true
-        schema:
-          type: integer
-          title: Location Id
-        name: location_id
-        in: path
-      - required: true
-        schema:
-          type: string
-          title: Dataset Id
-        name: dataset_id
-        in: path
-      - required: true
-        schema:
-          type: integer
-          exclusiveMinimum: true
-          title: User Id
-          minimum: 0
-        name: user_id
-        in: query
-      - description: Automatic directory expansion. This will be replaced by pagination
-          the future
-        required: false
-        schema:
-          type: boolean
-          title: Expand Dirs
-          description: Automatic directory expansion. This will be replaced by pagination
-            the future
-          default: true
-        name: expand_dirs
-        in: query
+        - required: true
+          schema:
+            type: integer
+            title: Location Id
+          name: location_id
+          in: path
+        - required: true
+          schema:
+            type: string
+            title: Dataset Id
+          name: dataset_id
+          in: path
+        - required: true
+          schema:
+            type: integer
+            exclusiveMinimum: true
+            title: User Id
+            minimum: 0
+          name: user_id
+          in: query
+        - description: Automatic directory expansion. This will be replaced by pagination the future
```

Comment:
this looks like a lot of space/line/quoting modifications for no real change. was something changed lately in how these openapi files are generated?